templates/CRM/Event/Page/EventInfo.tpl

```diff
@@ -102,11 +102,17 @@
       <div class="label">{ts}When{/ts}</div>
       <div class="content">
         {strip}
-            {$event.event_start_date|crmDate}
+            {if $event.event_start_date && $event.event_end_date && ($event.event_end_date|date_format:"%Y%m%d" == $event.event_start_date|date_format:"%Y%m%d")}
```

Comment:
Hi @shaneonabike We can't use smarty date_format anymore because it fails in php 8.1 because it uses strftime() which has been banished. Can you use crmDate?

Reply:
Ok that's done!